src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(initValue, argValue) {
+      return initValue + argValue;
+    },
```

Comment:
you can use result from calculator

```suggestion
    add(argValue) {
      return calculator.result + argValue;
    },
```